src/components/Register.js

```diff
@@ -0,0 +1,83 @@
+import { useState } from "react"
+import { TextField, Button } from "@mui/material"
+import { Link } from "react-router-dom"
+import '../styles/Register.css'
+
+const Register = () => {
+    
+    const [createdUser, setCreatedUser] = useState(false)
+    const [username, setUserName] = useState('')
+    const [email, setEmail] = useState('')
+    const [password, setPassword] = useState('')
+
+
+    const handleRegister = (event) => {
+        event.preventDefault()
+        if (username && email && password) {
+            const userObject = {
+                username,
+                email,
+                password
+            }
+            fetchRegister(userObject)
+        }
+    }
+
+    const fetchRegister = async (user) => {
+        const url = `https://recipe-blog-l7ey.onrender.com/recipe-blog/register`
+
+        const options = {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify({
+                'username': user.username,
+                'email': user.email,
+                'password': user.password
+            })
+        }
+
+        try {
+            const response = await fetch(url, options)
+
+            const data = await response.json()
+                
+            localStorage.setItem('token', data.token)
+            localStorage.setItem('user', data.user.id)
+
+            if (!response.ok) {
+                throw new Error(`Error: ${response.status}`)
+            }
+            setCreatedUser(true)
+        } catch (error) {
+            console.log(error)
+        }
+    }
+
+    
+    return (
+        <>
```

Comment:
Add the following code to display an error message if the error state is not null:
```
{errorMessage !== null && (
  <div>
    <h3 style={{textAlign: 'center', color: '#F00' }}>{errorMessage}</h3>
  </div>
)}
```